.github/CODEOWNERS

```diff
@@ -32,3 +32,6 @@ yarn.lock                                           @backstage/community-plugins
 /workspaces/tech-insights                           @backstage/community-plugins-maintainers  @xantier
 /workspaces/report-portal                           @backstage/community-plugins-maintainers  @yashoswalyo
 /workspaces/redhat-resource-optimization            @backstage/community-plugins-maintainers
+/workspaces/3scale                                  @backstage/community-plugins-maintainers  @fmenesesg
+/workspaces/redhat-argocd                           @backstage/community-plugins-maintainers  @karthikjeeyar @rohitkrai03 @Eswaraiahsapram
+/workspaces/scaffolder-relation-processor           @backstage/community-plugins-maintainers
```

Comment:
Should we also include Frank here since it's his original code? Or perhaps you since you're taking over ownership of it while he's back at Uni?

```suggestion
/workspaces/scaffolder-relation-processor           @backstage/community-plugins-maintainers  @Zaperex
```

Reply:
Not sure if the errors in this file are expected and some process will add people once the file is pushed. https://github.com/backstage/community-plugins/blob/main/CONTRIBUTING.md#manual-migration-steps

Reply:
For this to work your need to be an [Org Member](https://github.com/backstage/community/blob/main/GOVERNANCE.md#organization-member); ongoing conversation as to how we approach that for this repo is in https://github.com/backstage/community-plugins/issues/707

Reply:
> Should we also include Frank here since it's his original code? Or perhaps you since you're taking over ownership of it while he's back at Uni?

Included myself here after discussion with @Zaperex